.circleci/config.yml

```diff
@@ -0,0 +1,86 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
+    working_directory: ~/wp-calypso
+    docker:
+      - image: circleci/node:8.9.4-browsers
+        environment:
+          CIRCLE_ARTIFACTS: /tmp/artifacts
+          CIRCLE_TEST_REPORTS: /tmp/test_results
+
+    steps:
+      - checkout
+
+      - run:
+          name: Create Directories for Results and Artifacts
+          command: |
+            mkdir -p $CIRCLE_ARTIFACTS
+            mkdir -p $CIRCLE_TEST_REPORTS
+
+      - restore_cache:
+          key: v1-{{ .Branch }}-{{ checksum "package.json" }}
+
+      - run: npm install
+
+      - save_cache:
+          key: v1-{{ .Branch }}-{{ checksum "package.json" }}
+          paths:
+            - "node_modules"
+
+      - run: NODE_ENV=test npm run build-server
+
+      - run:
+          command: |
+            npm run translate; mkdir -p $CIRCLE_ARTIFACTS/translate
+            mv calypso-strings.pot $CIRCLE_ARTIFACTS/translate
+
+      - run:
+          command: |
+            git clone https://github.com/Automattic/gp-localci-client.git
+            bash gp-localci-client/generate-new-strings-pot.sh $CIRCLE_BRANCH $CIRCLE_SHA1 $CIRCLE_ARTIFACTS/translate
+            rm -rf gp-localci-client
+
+      - run:
+          name: Run Integration Tests
+          command: |
+            bin/run-integration $(circleci tests glob "bin/**/integration/*.js" "client/**/integration/*.js" "server/**/integration/*.js" | circleci tests split --split-by=timings)
+
+      - run:
+          name: Lint
+          command: npm run lint:config-defaults
+
+      - run:
+          name: Lint Client and Server
+          command: |
+            ./node_modules/.bin/eslint-eslines $(circleci tests glob "client/**/*.js" "client/**/*.jsx" "server/**/*.js" "server/**/*.jsx" | circleci tests split)
+
+      - run:
+          name: Run Client Tests
+          command: |
+              npm run test-client:ci $(circleci tests glob "client/**/test/*.js" "client/**/test/*.jsx" | circleci tests split --split-by=timings)
+
+      - run:
+          name: Run Server Tests
+          command: |
+              npm run test-server:ci $(circleci tests glob "server/**/test/*.js" "server/**/test/*.jsx" | circleci tests split --split-by=timings)
+
+      - run:
+          name: Gather Test Results
+          command: |
+            find . -type f -regex  ".*/test-results.*\.xml" -exec cp {} $CIRCLE_TEST_REPORTS/ \;
+          when: always
+
+      - store_test_results:
+          path: /tmp/test_results
+      - store_artifacts:
+          path: /tmp/test_results
+      - store_artifacts:
+          path: /tmp/artifacts
+
+# We need to work around this bit because
+# the notification system in 2.0 is a bit flakey
+# and is not yet working as expected.
+# notify:
+#   webhooks:
+#     - url: https://translate.wordpress.com/api/localci/-relay-new-strings-to-gh
```

Comment:
Isn't this a deal-breaker for now?

Reply:
hmm cc @Automattic/i18n

Reply:
For Team Global:
- Please refresh us on what this task does 😄 
- Can it be done at some interval, or does this need to happen on each build?

Reply:
This enables the a8ci18n bot messages about translation strings, and yep - it should run on each build.

Reply:
The public support docs seem to think that moving this to the top of the file may make it work...?

Reply:
actually, no. there are multiple examples of it working at the bottom.

https://discuss.circleci.com/t/circleci-2-0-notifications-webhooks/19075/7

Reply:
@Automattic/i18n @yoavf I turned this notification section back on in https://github.com/Automattic/wp-calypso/pull/23131/commits/a90405093ebde318959818e45cc8e826d12d5df9 a couple days ago. Can you confirm that the webhook fired and you were notified?

If not, we can re-run the build and see what happens. 

Thanks!